src/qt/rpcconsole.cpp

```diff
@@ -274,6 +277,19 @@ RPCConsole::RPCConsole(const PlatformStyle *platformStyle, QWidget *parent) :
     connect(ui->fontSmallerButton, SIGNAL(clicked()), this, SLOT(fontSmaller()));
     connect(ui->btnClearTrafficGraph, SIGNAL(clicked()), ui->trafficGraph, SLOT(clear()));
 
+    //Setup autocomplete and attach it
+    QStringList wordList;
+    for (int i =0; i < 53; ++i)
```

Comment:
@GamerSg Right, somehow forgot we are still waiting on travis to get c++11 rolled out.

You could take a look at `src/utilstrencodings.h` which has a `#define ARRAYLEN(array)`.